contrib/inventory/ec2.py

```diff
@@ -881,6 +882,16 @@ def add_instance(self, instance, region):
             if self.nested_groups:
                 self.push_group(self.inventory, 'instance_states', state_name)
 
+        # Inventory: Group by platform
+        if self.group_by_platform:
+            if instance.platform:
+                platform = self.to_safe('platform_' + instance.platform)
+            else:
+                platform = self.to_safe('platform_undefined')
+            self.push(self.inventory, platform, dest)
```

Comment:
I cannot add commits to this PR to fix this myself, so I'm updated this to request changes.